go/pkg/yolosvc/api_buildlist_test.go

```diff
@@ -0,0 +1,82 @@
+package yolosvc
+
+import (
+	"context"
+	"testing"
+
+	"berty.tech/yolo/v2/go/pkg/testutil"
+	"berty.tech/yolo/v2/go/pkg/yolopb"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBuildList(t *testing.T) {
```

Comment:
```suggestion
func TestService_BuildList(t *testing.T) {
```

to follow the go test funcs naming convention